File: src/sdk/php/3/core-classes/security/scroll-users/snippets/scroll-users-1.php

<?php

use \Kuzzle\Kuzzle;

$scrollId = 'myScrollId';
$options = [];

$kuzzle = new Kuzzle('localhost');

try {
  $kuzzle->security()->scrollUsers($profileId, $options);
}
catch (ErrorException $e) {
  // Handle error
}

File: src/sdk/java/2/core-classes/security/scroll-users/snippets/scroll-users-1.java

kuzzle
  .security
  .scrollUsers(scrollId, options, new ResponseListener<SecurityDocumentList>() {
    @Override
    public void onSuccess(SecurityDocumentList response) {
      // called once the scroll action has been completed
    }

    @Override
    public void onError(JSONObject error) {
      // Handle error
    }
  });
